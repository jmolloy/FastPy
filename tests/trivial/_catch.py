try:
    print x
except NotImplementedError, e:
    print e
except RuntimeError as e:
    print "e is"
    print e

