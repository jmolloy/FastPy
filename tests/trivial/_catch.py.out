e is
Key does not exist: "x"
